### ISSUE TITLE
Attempting to duplicate a compendium results in `TypeError: pack.duplicate is not a function`

### ISSUE THREAD

**Issue by Varriount** [earlSt1/vtt-compendium-folders#88]
Foundry Version: 0.8.8
Compendium Folders Version: 2.3.2

When attempting to duplicate a compendium placed inside a folder, an error is thrown with the following stack trace:
```
yes https://<snip>/modules/compendium-folders/compendium-folders.js:836
    callback https://<snip>/scripts/foundry.js:20505
    submit https://<snip>/scripts/foundry.js:20449
    _onClickButton https://<snip>/scripts/foundry.js:20412
    jQuery 2
        dispatch
        handle
```

This isn't too major (duplication still works if I temporarily disable Compendium Folders) but it is annoying. I'm aware that there's been a couple of changes to how indexes work across 0.8.x, so that may be the cause.

**Comment by earlSt1** [comment]
Hi thanks for the report, which game system + version are you using?

**Comment by Varriount** [comment]
Oh! Sorry, I completely forgot to give those details.

I'm running D&D 5e, version 1.3.6

**Comment by earlSt1** [comment]
Sorry i completely forgot i already had this in my test version. It should be fixed in v2.3.3 if you update
